src/wallet/components/ConnectAccount.tsx

```diff
@@ -1,56 +0,0 @@
-import { useCallback } from 'react';
-import { useAccount, useConnect, useDisconnect } from 'wagmi';
-import type { ConnectAccountReact } from '../types';
-
-/**
- * ConnectAccount
- *  - Connects to the wallet
- *  - Disconnects from the wallet
- *  - Displays the wallet network
- */
-export function ConnectAccount({ children }: ConnectAccountReact) {
```

Comment:
This, will cause a breaking changes. 🤔

Which can be tricky as ConnectAccount is already popular as component. 🤔

Let's avoid this breaking changes for now. So we still keep `ConnectAccount` in the codebase, BUT we remove it from the documentation. So we gently showcase the new one instead, and then we will kill `ConnectAccount` in a three of weeks. After the new one has been already around.

So we avoid a drastic change, but we show the new first, we hide the old, and then when people forgot about the old we kill it.